app/server/models/ussd.py

```diff
@@ -19,6 +19,19 @@ class UssdMenu(ModelBase):
     display_text_en = db.Column(db.String, nullable=False)
     display_text_sw = db.Column(db.String, nullable=False)
 
+    @staticmethod
+    def find_by_name(name: str) -> "UssdMenu":
+        menus = UssdMenu.query.filter_by(name=name)
+        if menus.count() == 0:
+            sentry.captureMessage("No USSD Menu with name {}".format(name))
```

Comment:
if we want to gracefully fail and log the issue, should we be using sentry?

Reply:
Yep, exactly